spec/lib/rbac/query_shared_resource_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe RBAC::QuerySharedResource do
+  let(:app_name) { 'catalog' }
+  let(:resource) { "portfolios" }
+  let(:verbs) { ["read"] }
+  let(:resource_id1) { "1" }
+  let(:role1_uuid) { "1" }
+  let(:role2_uuid) { "2" }
+  let(:group1) { double(:name => 'group1', :uuid => "123") }
+  let(:group2) { double(:name => 'group2', :uuid => "12345") }
+  let(:group3) { double(:name => 'group3', :uuid => "45665") }
+  let(:role1) { double(:name => 'role1-Sharing', :uuid => role1_uuid) }
+  let(:role2) { double(:name => 'role2-Sharing', :uuid => role2_uuid) }
+  let(:role1_detail) { double(:name => 'role1-Sharing', :uuid => role1_uuid, :access => [access3]) }
+  let(:role2_detail) { double(:name => 'role2-Sharing', :uuid => role2_uuid, :access => [access1, access2]) }
+  let(:groups) { [group1, group2, group3] }
+  let(:roles) { [role1, role2] }
+  let(:policy1) { double(:group => group1, :roles => roles) }
+  let(:policies) { [policy1] }
```

Comment:
This can be condensed to
```
let(:policies) { [double(:group => group1, :roles => roles)] }
```

Since `policy1` isn't used elsewhere in this file.